crates/services/p2p/Cargo.toml

```diff
@@ -50,6 +50,7 @@ thiserror = "1.0.47"
 tokio = { workspace = true, features = ["sync"] }
 tracing = { workspace = true }
 void = "1"
+hickory-resolver = "0.25.0-alpha.2"
```

Comment:
Can we use resolver from `libp2p`?

Reply:
do you mean the same version as libp2p or to use their resolver?

their resolver is in libp2p-dns (https://docs.rs/libp2p-dns/latest/libp2p_dns/tokio/type.Transport.html) and doesn't export any helper functions to resolve dnsaddr's.

Reply:
I see, they use the same library to do it. Yeah. we need to use the same version to not bloat the `Cargo.lock`

Reply:
addressed in [4f4b662](https://github.com/FuelLabs/fuel-core/pull/2204/commits/4f4b662c4f9c6e0ac22d381075c58a58723329a5)